apps/api/src/app/environments-v1/usecases/construct-framework-workflow/construct-framework-workflow.usecase.ts

```diff
@@ -2,22 +2,32 @@ import { Injectable, InternalServerErrorException } from '@nestjs/common';
 import {
```

Comment:
@tatarco Should we add these use cases to environment-v1 or environment-v2 path? I'd assume the latter.

Reply:
We are saying pushing this is urgent, this is preexisting code, can be changed in a future PR, I personally don't think it matters at all, but naming is not a passion of mine, I use the IDE to find whatever I need